shared/reducers/teams.tsx

```diff
@@ -217,6 +217,30 @@ export default (
           () => RPCChatTypes.ConversationMemberStatus.left
         )
         return
+      case TeamsGen.setTeamRoleMapLatestKnownVersion: {
+        let latestKnownVersion = action.payload.version
+        if (latestKnownVersion > draftState.teamRoleMap.latestKnownVersion) {
+          draftState.teamRoleMap = {
```

Comment:
I think you can just `draftState.teamRoleMap.latestKnownVersion = latestKnownVersion`.